tests/unit/amo/sagas/testReviews.js

```diff
@@ -1,7 +1,7 @@
 import { hideLoading, showLoading } from 'react-redux-loading-bar';
 import SagaTester from 'redux-saga-tester';
 
-import * as amoApi from 'amo/api';
+import * as reviewsApi from 'amo/api/reviews';
```

Comment:
Thanks for moving those API methods to their own folder btw, it makes them a lot easier to sort through 😄